gcp/gcp_instance.go

```diff
@@ -20,6 +20,12 @@ func (p *GCloud) CreateInstance(ctx *lepton.Context) error {
 		c.CloudConfig.Flavor = "g1-small"
 	}
 
+	// short-circuit to instancegroup
+	if c.RunConfig.InstanceGroup != "" {
+		p.addToInstanceGroup(ctx, c.RunConfig.InstanceGroup)
+		return nil
```

Comment:
It seem this would work, but any firewall rules that may be necessary to open TCP and UDP ports specified in the config wouldn't be created, and if the config specifies a domain name the relevant DNS records wouldn't be created either.

Reply:
domains would not be attached to instances in the instance group - it acts as an auto-scaling load balancer

Reply:
added ports for instance template